pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<groupId>uk.ac.ucl.rc.development.oracc</groupId>
 	<artifactId>nammu</artifactId>
-	<version>0.6.0</version>
```

Comment:
Since there's at least a new feature (the clickable errors) and not just bugfixes of 0.6.0, I think I'd move onto `0.7.0`.